.github/workflows/pr_build.yml

```diff
@@ -68,6 +68,26 @@ jobs:
         run: |
           cd examples/powertools-examples-core/gradle
           ./gradlew build
+      - name: Setup Terraform
+        if: ${{ matrix.java == '11' }}
+        uses: hashicorp/setup-terraform@633666f66e0061ca3b725c73b2ec20cd13a8fdd1 #v2.0.3
+      - name: Terraform validate
+        if: ${{ matrix.java == '11' }}
+        run: |
+          terraform -version
+          cd examples/powertools-examples-core/terraform
+          terraform init
+          terraform validate
```

Comment:
should we do a `plan` too ?

Reply:
Your call, if you think it adds value to the build.

Reply:
terraform plan was added.

Reply:
maybe because you don't have permission to do that as it's from a fork